ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/logging.j2

```diff
@@ -71,6 +71,9 @@ logging format hostname ipv4
 {%     if logging.format.sequence_numbers is arista.avd.defined(true) %}
 logging format sequence-numbers
 {%     endif %}
+{%     if logging.format.rfc5424 is arista.avd.defined(true) %}
+logging format rfc5424
```

Comment:
Please make sure the order is correct according to how EOS renders the config. It is probably correct already, but just confirm please :)

Reply:
I thought my comment had made followed the resolution:
Ah, of course! I just confirmed that it wasn't in it's rendered position - I'll move it!

A question, should I change schemas to reflect this?

Reply:
Schema order does not have to match the rendered config order. The order in the schema should follow whatever makes sense for the docs.